assets/src/edit-story/components/library/panes/text/textSets/textSet.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { rgba } from 'polished';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import DisplayElement from '../../../../canvas/displayElement';
+import { PAGE_WIDTH, TEXT_SET_SIZE } from '../../../../../constants';
+import useBatchingCallback from '../../../../../utils/useBatchingCallback';
+import useLibrary from '../../../useLibrary';
+
+const TextSetItem = styled.div`
```

Comment:
I was actually thinking of the whole accessibility and keyboard navigation being part of the panel issue (#4080) since that's where it all was supposed to come together in the panel.

However, I can def start adding things before already (In #4478 the [container is a "group"](https://github.com/google/web-stories-wp/pull/4478/files#diff-9ea906a3374da5d2bf67d69b5302ca9eR52) and each [item is a "listitem"](https://github.com/google/web-stories-wp/pull/4478/files#diff-aadd9ee462ae026aaae26c4c4d365ef6R79))

WDYT?

Reply:
(Changed to button and added a label.)

Reply:
That's fine with me as long as it's addressed 👍